app/src/main/java/com/sdp13epfl2021/projmag/model/TagsBaseManager.kt

```diff
@@ -91,4 +91,13 @@ import java.util.regex.Pattern
          return listString
      }
 
+     fun isListOfStringsValidTags(listTags : List<String>) : Boolean{
+         for(tag in listTags){
+             if(!tags.contains(Tag(tag))){
+                 return false
+             }
+         }
+         return true
+     }
```

Comment:
This could be simplified by 
```suggestion
         listTags.map(::Tag).all(tags::contains)
```

Reply:
Same reason as the other one line / multilines ?